src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/IepSummary.kt

```diff
@@ -53,11 +56,6 @@ data class IepSummary(
   }
 }
 
-fun daysSinceReview(iepHistory: List<IepDetail>): Int {
-  val currentIepDate = LocalDate.now().atStartOfDay()
-  return Duration.between(iepHistory.first().iepDate.atStartOfDay(), currentIepDate).toDays().toInt()
```

Comment:
`iepHistory` is an empty list when `IepSummary` is loaded without details: this would have thrown an exception. as would the method, but not the field!

Reply:
I didn't check, but when getting this without details, is the `iepDate` field just set to the latest IEP date somehow?